modyn/supervisor/internal/evaluation_result_writer/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""Supervisor module. The supervisor initiates a pipeline and coordinates all components.
```

Comment:
copy & pasted docstring :)

Reply:
Yes, I see. But note that this docstring is used by every submodule of the supervisor (e.g supervisor/internal/triggers or /utils)